omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -73,6 +73,12 @@ http {
   }
   <%- end -%>
 
+  <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token'] -%>
+  upstream data-collector {
+    server <%= URI.parse(node['private_chef']['data_collector']['root_url']).host %>:<%= URI.parse(node['private_chef']['data_collector']['root_url']).port %>;
```

Comment:
Ok, I just wanted to be extra certain about this, since we would have to change the user-facing config from a string to an array which would make upgrading painful.